scripts/install-chef-suse.sh

```diff
@@ -291,7 +291,7 @@ function reset_crowbar()
 {
     rm -f $crowbar_install_dir/crowbar-install-failed
     rm -f $installation_steps
-    su -s /bin/sh - crowbar sh -c "cd /opt/dell/crowbar_framework && RAILS_ENV=production bin/rake db:cleanup"
+    su -s /bin/sh - crowbar sh -c "cd /opt/dell/crowbar_framework && RAILS_ENV=production bin/rake db:cleanup db:seed"
```

Comment:
`db:cleanup` resets the complete database which means that we loose all the data including the default values. ATM we don't have any default values but after the switch to the Node in the DB the admin node will be such a default value.